addons/mail/static/src/discuss/call/common/call.scss

```diff
@@ -40,6 +40,10 @@
     aspect-ratio: 16/9;
 }
 
+.o-discuss-Call-insetCard {
+    background-color: black;
+}
+
```

Comment:
There's a `.o-bg-black` that is theme-independent: https://github.com/odoo/odoo/blob/17.0/addons/mail/static/src/core/common/core.scss#L1
Could use that instead of defining in CSS, so less LOCs.